Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/qrcode/TraceLocation.kt

```diff
@@ -34,34 +35,25 @@ data class TraceLocation(
 fun List<TraceLocationEntity>.toTraceLocations() = this.map { it.toTraceLocation() }
 
 fun TraceLocationEntity.toTraceLocation() = TraceLocation(
-    guid = guid,
+    id = id,
     type = type,
     description = description,
     address = address,
     startDate = startDate,
     endDate = endDate,
     defaultCheckInLengthInMinutes = defaultCheckInLengthInMinutes,
-    byteRepresentation = byteRepresentationBase64.decodeBase64()!!,
-    signature = signatureBase64.decodeBase64()!!,
+    cryptographicSeed = cryptographicSeedBase64.decodeBase64()!!,
```

Comment:
unrelated to your changes but why are we decoding base64 here instead of having a room type converter like for Instants

Reply:
Good point! Should do a refactoring at some point.